schemas/imaging/acquisition.json

```diff
@@ -49,6 +49,95 @@
 			"format": "date-time",
 			"title": "Session end time"
 		},
+		"local_storage_directory": {
+			"type": "string",
+			"title": "Local storage directory"
+		},
+		"external_storage_directory": {
+			"type": "string",
+			"title": "External storage directory"
+		},
+		"x_axis": {
```

Comment:
to do this unambiguously you also have to indicate "which dimension is x/y/z".   So, some sort of x=0, y=1, z=2 or something.